### ISSUE TITLE
[Feature Request] Multiple Tabs

### ISSUE THREAD

**Issue by MTecknology** [julian-alarcon/prospect-mail#185]
Please disregard if there's any reason this shouldn't happen, but... one feature I miss from thunderbird is the ability to keep the calendar and mail in different tabs so that they can be easily switched between without losing position or waiting for a page reload. I also sometimes find it helpful for long-winded email threads.

**Comment by ErwinBaeyens** [comment]
Yes this would be a great addition, more so since it seems to be impossible to run 2 instances of the app as well.

**Comment by tgrzegorzewski** [comment]
Would love to see this feature as well. If I try to open Calendar in a new window it just uses the current window.
